### ISSUE TITLE
typical texture loss values

### ISSUE THREAD

**Issue by roystonrodrigues** [hhb072/WaveletSRNet#2]
What are the typical texture loss values while training... I have done my own implementation of the above network.. the texture loss has a torch.mean its value to 0.0005 range.. where as other losss sr loss, lr loss and full image loss have a torch.sum() component.. and are just divided by the batch size... they are in 10^3 range the total loss is dominated by sr loss,lr loss and full image loss. I dont understand how the texture loss is going to contribute towards the learning.
